### DISCUSSION TITLE
Would love help to go through issues

### DISCUSSION THREAD

**Discussion by AntonOsika** [AntonOsika/gpt-engineer#472] [Announcements]
We have the ambition to keep the repo clean 🚀

Those that want to help us build the open platform for devs to tinker with and build their personal AI programmer toolbox:

If you go through a issues without a clear resolution, make up your mind, and tag me or @patillacode with your recommendation on if we should close the issue or do something else that would be very valuable 🙏 

Much more so than submitting small PRs with minor fixes at the moment (although that is also welcome)

Thanks!

**Comment by geewhizbang** [comment]
I would like to help with rewriting instructions, especially with respect to installing on windows, and to fill in the details a bit better for newbies like me.

**Reply by AntonOsika** [reply]
Awesome.

Do you want to DM me in discord about this?

**Reply by AntonOsika** [reply]
Feel free to get started already and I'll see you there

**Comment by mentallysane19** [comment]
Intresting
